spec/controllers/groups_controller_spec.rb

```diff
@@ -0,0 +1,28 @@
+require "rails_helper"
+
+RSpec.describe GroupsController do
+  describe "GET /groups" do
+    context "when invalid params" do
+      it "does not create a new group" do
+        expect do
+          post :create, params: { group: { name: "St. Paul" } }
+        end.not_to change(Group, :count)
+      end
+
+      it "sets an alert message" do
+        post :create, params: { group: { name: "St. Paul" } }
+
+        expect(flash[:alert])
+          .to match(
```

Comment:
The `.` should be in the end of the previous line. Ref: https://github.com/thoughtbot/guides/blob/master/style/ruby/sample.rb#L11

But maybe change this to be:
```
expect(flash[:notice]).to match(
  I18n..t("flash.actions.create.alert", resource_name: "Group")
)
```

Reply:
Done with fixup in `479661d`